### DISCUSSION TITLE
aleph dev throws error!!

### DISCUSSION THREAD

**Discussion by sahilrajput03** [alephjs/aleph.js#329] [Q&A]
![image](https://user-images.githubusercontent.com/31458531/119261032-11804c80-bbf3-11eb-8da4-ce4331bacee5.png)

As you can see all the details you need, please suggest why is it not working..??

**Comment by sahilrajput03** [comment]
https://github.com/sahilrajput03/hello 
This is the repo link though..

**Comment by thomascarvalho** [comment]
Hi, Alephjs doesn't work with deno 1.7+ version.

Use the latest alpha version of aleph ( `deno run -A https://deno.land/x/aleph/install.ts`) or downgrade deno.

**Comment by sahilrajput03** [comment]
![image](https://user-images.githubusercontent.com/31458531/119267192-1fda6280-bc0b-11eb-8e96-df9f813d4133.png)

I changed my deno installation as you can see that!! But how do I remove aleph and again install aleph if thats what I am missing now..?

**Comment by sahilrajput03** [comment]
@thomascarvalho ?

**Comment by thomascarvalho** [comment]
I am using deno 1.10.2, I don't know how to specifically respond to this error.

What I advise you (this is the first solution that I proposed to you) is to keep Deno in version 1.10.2 and install the last alpha version of alephjs.

All you have to do
`deno upgrade`
and
`deno run -A https://deno.land/x/aleph/install.ts`

then start a new project with the init command

**Comment by sahilrajput03** [comment]
I will be back with a try of this solution as well..!.

**Comment by sahilrajput03** [comment]
![image](https://user-images.githubusercontent.com/31458531/119269514-b3b12c00-bc15-11eb-8ad7-815aae432f47.png)

Okay, its still very same.. !
:( what is wrong with my machine or deno or alephjs. ..?
@thomascarvalho

**Comment by sahilrajput03** [comment]
![image](https://user-images.githubusercontent.com/31458531/119269601-30440a80-bc16-11eb-8da1-5631c357e53c.png)

**Comment by thomascarvalho** [comment]
I don't know why you have this error.

Try with --reload flag
`deno run --reload -A https://deno.land/x/aleph/install.ts`

**Comment by sahilrajput03** [comment]
![image](https://user-images.githubusercontent.com/31458531/119270033-58346d80-bc18-11eb-8875-5294eb6419be.png)
Amazing!!! 🤺︎🤺︎

**Comment by sahilrajput03** [comment]
oh noo!! :(
![image](https://user-images.githubusercontent.com/31458531/119270058-7a2df000-bc18-11eb-88e6-4af62df2288b.png)

**Reply by sahilrajput03** [reply]
It still doesn't work why..?

**Reply by sahilrajput03** [reply]
@thomascarvalho any possible thing i should try now..?

**Reply by thomascarvalho** [reply]
Maybe It's a cache problem, again.

I don't know, but you can try
`deno cache --reload https://deno.land/x/aleph@v0.3.0-alpha.33/cli.ts`
or `deno cache --reload https://deno.land/x/aleph@v0.3.0-alpha.33/cli/init.ts`

and retry the init.

**Reply by sahilrajput03** [reply]
Its so amazing to see how deno just depends on cache for everything..! 
It works now!!

![image](https://user-images.githubusercontent.com/31458531/119272783-5fae4380-bc25-11eb-82ea-0a7c6e53344a.png)

Thanks buddy, it works now finally with latest deno and latest alephjs!!
